init_single_platform/after/SampleApp.xcodeproj

```diff
@@ -44,15 +44,15 @@ Targets:
           GCC_PRECOMPILE_PREFIX_HEADER: 'YES'
           GCC_PREFIX_HEADER: SampleApp/SampleApp-Prefix.pch
           INFOPLIST_FILE: SampleApp/SampleApp-Info.plist
-          PRODUCT_NAME: $(TARGET_NAME)
+          PRODUCT_NAME: "$(TARGET_NAME)"
```

Comment:
revert them. I am fairly certain you only need `.sh` file changes based of your shell script change PR.